File: CPE-1star/UVA10783-Odd Sum .py
#第一個輸入為幾組，每兩個數字一組，計算這兩個內奇數的和
while True:
	try:
		n=int(input())
		count=1
		for i in range(n):
			a=int(input())
			b=int(input())
			temp=0
			for j in range(a,b+1):
				if j%2!=0:
					temp+=j
			print("Case %d: %d"%(count,temp))
			count+=1
	except:
		break

File: CPE-1star/UVA10783-Odd Sum .cpp
#include <iostream>
using namespace std;
 
int main()
{
    int n,count=1;
    while(cin>>n){
        int num[2];
        for(int i=0;i<n;i++){
            cin>>num[0];
            cin>>num[1];
            int temp=0;
            for(int j=num[0];j<=num[1];j++)
                if(j%2!=0)
                    temp+=j;
            cout<<"Case "<<count<<": "<<temp<<'\n';
            count+=1;
        }
    }
    return 0;
}
